.github/workflows/ci.yml

```diff
@@ -17,9 +17,12 @@ jobs:
     name: Lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - uses: swatinem/rust-cache@81d053bdb0871dcd3f10763c8cc60d0adc41762b # pin@v1
+      - name: Install Rust Toolchain
+        run: rustup toolchain install stable --profile minimal --component clippy rustfmt
```

Comment:
Does this line look correct? I referenced [this line](https://github.com/getsentry/relay/blob/c5c4b0561bb174739d87c1b9e69ac73197bc2321/.github/workflows/ci.yml#L39) from Relay and the old config changed in #1934

Reply:
there is also `--no-self-update`, but I believe that is most relevant for Windows.

You can take a look at https://github.com/getsentry/ophio/blob/c8b08063562f67306b6cc22a36322806fe8e588c/.github/workflows/ci.yml#L19-L25, which is the "most recent" version of setting all this up.

Also, `checkout` is already at `v4` as well :-)